### ISSUE TITLE
Inconsistency in capitalisation of ID

### ISSUE THREAD

**Issue by Resonance1584** [jenjinstudios/jira-connector#1]
Some options use ID and some use Id

e.g.

```
     * @method getWorkLog
     * @memberOf IssueClient#
     * @param {Object} opts The options to pass to the API.  Note that this object must contain EITHER an issueID or
     *     issueKey property; issueID will be used over issueKey if both are present.
     * @param {string} [opts.issueID] The ID of the issue.  EX: 10002
     * @param {string} [opts.issueKey] The Key of the issue.  EX: JWR-3
     * @param {string} opts.worklogId The id of the work log to retrieve.
```

issueID vs worklogId

**Comment by floralvikings** [comment]
Good catch; they should all probably be ```Id``` as that's the convention the Jira API documentation uses.  I'll try and get them all switched over this weekend.  Unfortunately, this is going to break backwards compatibility for anyone already using the capital ```ID```.

**Comment by floralvikings** [comment]
It is done; published as version 1.0.0.  Incremented major version to indicate a break in backwards compatibility.
